doc/developer-guides/hld/hv-startup.rst

```diff
@@ -111,70 +112,71 @@ Memory
 
 -  **Create VM**: A VM structure is allocated and initialized. A unique
    VM ID is picked, EPT is initialized, e820 table for this VM is prepared,
    I/O bitmap is set up, virtual PIC/IOAPIC/PCI/UART is initialized, EPC for
    virtual SGX is prepared, guest PM IO is set up, IOMMU for PT dev support
    is enabled, virtual CPUID entries are filled, and vCPUs configured in this VM's
-   ``vm config`` are prepared. For post-launched User VM, the EPT page table and
-   e820 table is actually prepared by DM instead of hypervisor.
+   ``vm config`` are prepared. For a post-launched User VM, the EPT page table
+   and e820 table are prepared by the DM instead of the hypervisor.
 
--  **Prepare vCPUs:** Create the vCPUs, assign the physical processor it
-   is pinned to, a unique-per-VM vCPU ID and a globally unique VPID,
-   and initializes its virtual lapic and MTRR, and its vCPU thread object got setup
-   for vcpu scheduling. The vCPU number and affinity are defined in corresponding
+-  **Prepare vCPUs:** Create the vCPUs, assign the physical processor the vCPU
+   is pinned to, a unique-per-VM vCPU ID and a globally unique VPID, initialize
+   its virtual lapic and MTRR, and set up its vCPU thread object for vCPU
+   scheduling. The vCPU number and affinity are defined in the corresponding
    ``vm config`` for this VM.
 
--  **Build vACPI:** For the Service VM, the hypervisor will customize a virtual ACPI
-   table based on the native ACPI table (this is in the TODO).
-   For a pre-launched VM, the hypervisor will build a simple ACPI table with necessary
-   information like MADT.
-   For a post-launched User VM, the DM will build its ACPI table dynamically.
+-  **Build vACPI:** For the Service VM, the hypervisor customizes a virtual ACPI
+   table based on the native ACPI table (this is in the TODO). For a
+   pre-launched User VM, the hypervisor builds a simple ACPI table with
+   necessary information such as MADT. For a post-launched User VM, the DM
+   builds its ACPI table dynamically.
 
--  **SW Load:** Prepares for each VM's SW configuration according to guest OS
-   requirement, which may include kernel entry address, ramdisk address,
-   bootargs, or zero page for launching bzImage etc.
-   This is done by the hypervisor for pre-launched or Service VM, and the VM will
-   start from the standard real or protected mode which is not related to the
-   native environment. For post-launched VMs, the VM's SW configuration is done
-   by DM.
+-  **SW Load:** Prepare for each VM's SW configuration according to guest OS
+   requirements, which may include kernel entry address, ramdisk address,
+   bootargs, or zero page for launching bzImage. This is done by the hypervisor
+   for pre-launched User VMs or Service VM, and the VM will start from the
+   standard real mode or protected mode which is not related to the native
+   environment. For post-launched User VMs, the VM's SW configuration is done by
+   DM.
```

Comment:
Optional: "SW" --> "software". ("HW" and "SW" seem like the author was just in a hurry.) Also, suggested punctuation changes: "This is done by the hypervisor for pre-launched User VMs or Service VM. The VM will start from the standard real mode or protected mode, which is not related to the native environment."